device/dhcp/servers/dhcpserver.yaml

```diff
@@ -8,9 +8,16 @@ components:
         ipv4_interfaces:
           description: >-
             This contains an array of references to IPv4 interfaces, each of which
-            will contain one DHCPv4 server.
+            will have list of DHCPv4 servers to different destinations.   
           type: array
-          maxItems: 10
           items:
             $ref: './v4/dhcpv4server.yaml#/components/schemas/DhcpServer.V4'
-          x-field-uid: 2
\ No newline at end of file
+          x-field-uid: 2
+        ipv6_interfaces:
+          description: >-
+            This contains an array of references to IPv6 interfaces, each of which
```

Comment:
Not clear on how one IPv4 interface can be configured with multiple DHCP servers.

Reply:
updatedd